components/html-rendering/src/main/java/org/datacleaner/result/html/HtmlAnalysisResultWriter.java

```diff
@@ -237,7 +236,7 @@ protected void writeBody(final Writer writer, final Map<ComponentJob, HtmlFragme
                     final ComponentDescriptor<?> descriptor = componentJob.getDescriptor();
                     if (!descriptor.equals(lastDescriptor)) {
                         final ComponentDocumentationWrapper wrapper = new ComponentDocumentationWrapper(descriptor);
-                        final String iconSrc = wrapper.getIconSrc(IconUtils.ICON_SIZE_MEDIUM);
+                        final String iconSrc = wrapper.getIconSrc(22);
```

Comment:
Magic number

Reply:
Yes but I was worried about using non-magic numbers here because I saw that depending on the screen size, the generated HTML would come out different. And I don't think we should change anything related to HTML generation in this round.

Reply:
Fair enough